### DISCUSSION TITLE
Changing the architecture of yolov8

### DISCUSSION THREAD

**Discussion by shivonafern** [ultralytics/ultralytics#4798] [General]
Hello I'm new to yolov8 so sorry if my question may sound basic... I want to add/remove few conv layers from the backbone of yolov8 how can I do so? Is it possible on colab? how can I override the custom yolov8.yaml file

**Comment by glenn-jocher** [comment]
@shivonafern hello! No worries, we all start somewhere, and it's great to see your interest in customizing YOLOv8. 😊

Yes, you can modify the architecture of YOLOv8, including adding or removing convolutional layers from the backbone. This is done by editing the model configuration file, typically a YAML file that defines the layers and their connections.

To make these changes, you would:
1. Download the default YOLOv8 YAML file.
2. Modify the file to add or remove layers as needed.
3. Save your custom YAML file.
4. Use this file when initializing your model in Colab.

When you start your training process, you'll specify the path to your custom YAML file so that YOLOv8 knows to use your modified architecture.

For detailed instructions on how to work with configuration files and customize models, please refer to our documentation at https://docs.ultralytics.com. You'll find guidance on model customization under the 'Train' section.

Happy coding, and best of luck with your YOLOv8 project! 🚀
